cilium/cmd/bpf_metrics_list.go

```diff
@@ -0,0 +1,61 @@
+// Copyright 2018 Authors of Cilium
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package cmd
+
+import (
+	"os"
+
+	"github.com/cilium/cilium/common"
+	"github.com/cilium/cilium/pkg/command"
+
+	"fmt"
+	"github.com/cilium/cilium/pkg/maps/datametrics"
+	"github.com/spf13/cobra"
+)
+
+const (
+	// DropForward is the bpf map key
+	DropForward = "KEY"
+	count       = "COUNT"
+)
+
+var bpfMetricsListCmd = &cobra.Command{
+	Use:   "metrics list",
```

Comment:
Please create a separate file, `cilium/cmd/bpf_metrics.go` which just adds a subcommand, then change this to "list". Compare against eg the recent `cilium bpf ipcache ...` changes. Otherwise the help messages and so on don't work correctly.

Reply:
will do

Reply:
done